src/PowerShellEditorServices/Workspace/ScriptRegion.cs

```diff
@@ -83,10 +83,21 @@ public sealed class ScriptRegion : IScriptExtent
         /// </returns>
         public static ScriptRegion Create(IScriptExtent scriptExtent)
         {
+            // IScriptExtent throws an ArgumentOutOfRange exception if Text is null
+            string scriptExtentText;
+            try
+            {
+                scriptExtentText = scriptExtent.Text;
+            }
+            catch (ArgumentOutOfRangeException e)
+            {
+                scriptExtentText = "";
```

Comment:
One general "best practice" in C# is when you need to use an empty string, use `string.Empty` instead of `""`.  I think it might save an allocation, but it's not that big of a deal if you don't do it.